contents/docs/api/user.mdx

```diff
@@ -14,14 +14,14 @@ This includes all possible event names, event properties, API key, app URLs, cur
 
 
 ```shell
-GET /user
+GET /users/@me
```

Comment:
One final change I think is worth making is adding a deprecation notice of `/api/user` somewhere, mentioning the version where it was deprecated and that support will be removed soon.

Reply:
you mean in addition to the output that we already see from `/api/user/`?

> "deprecation": "Endpoint has been deprecated. Please use /api/v2/user/

which I mentioned in the description I'll change to be `/api/users/@me/`

Reply:
Yeah I meant more like in these docs.